src/components/study-pane.js

```diff
@@ -689,9 +711,22 @@ const StudyPane = (props) => {
     useEffect(() => {
         websocketExpectedCloseRef.current = false;
         dispatch(openStudy(studyUuid));
-        loadNetwork();
-        loadGeoData();
-        loadTree();
+
+        fetchStudyExists(studyUuid).then((response) => {
```

Comment:
Why not use `checkStudyExists` here ? which is no more used otherwise... It's useEffect dependency has no meaning anymore.
Remove checkStudyExists method or make it return a promise and manage reponse here ?

Reply:
forgot to clean up that unused function indeed